### ISSUE TITLE
Show project name on invoice https://app.invoicing.co/#/invoices

### ISSUE THREAD

**Issue by theoryshaw** [invoiceninja/ui#2058]
This is a UI comment. 
It seems the `project name` should be somewhere in this area. 

![image](https://github.com/user-attachments/assets/19769ba0-7c6a-4386-b1fe-bbcda5905e1c)

**Comment by turbo124** [comment]
@Civolilah if project is set, diplay standard label + navigation link to  project

**Comment by theoryshaw** [comment]
Did this get pushed to production yet?

**Comment by Civolilah** [comment]
> Did this get pushed to production yet?

@theoryshaw I just tested this in production and it works correctly. I created a test project with some tasks, triggered the "Invoice Project" action, and after saving the invoice, the label appeared on the edit invoice page. Screenshot:

<img width="512" alt="Screenshot 2024-11-21 at 07 24 03" src="https://github.com/user-attachments/assets/d251ff9a-ec1e-4fe4-91bd-2b99e68eccb6">

Let me know if you find any issues.
